e2e_tests/tests/cli.test.js

```diff
@@ -22,8 +22,8 @@ describe('KYT CLI', () => {
   });
   it('sets up a starter-kyt', () => {
     let setupURL = 'git@github.com:NYTimes/kyt-starter-test.git';
-    if (process.env.TEST_TOKEN) {
-      setupURL = `https://${process.env.TEST_TOKEN}@github.com/NYTimes/kyt-starter-test.git`;
+    if (process.env.CI) {
+      setupURL = 'https://github.com/NYTimes/kyt-starter-test.git';
```

Comment:
Does this URL not work for local e2e testing? E.g. why does it need to be specific to CI?